docs/guide/algob-console.md

```diff
@@ -10,127 +10,357 @@ Sometimes it's nice to work with your contracts interactively for testing and de
+Here we will transfer some `algos` from an algorand smart contract ([`/assets/teal/2-gold-contract-asc.teal`](https://github.com/scale-it/algo-builder/blob/develop/examples/asa/assets/teal/2-gold-contract-asc.teal)) to `john`.
++ We will first load the logic signature (using `deployer.loadLogic(<file_name>.teal)` and get it's address(`lsig.address()`).
++ This address will act as the sender(contract account mode) and receiver is `john`.
++ Finally, we transfer some algos using `algob.executeTransaction(..)` function. Transaction will pass/fail according to asc logic.
 ```
 lsig = await deployer.loadLogic("2-gold-contract-asc.teal");
 sender = lsig.address();
 ```
 ![image](https://user-images.githubusercontent.com/33264364/97818537-e3740300-1cc8-11eb-81cd-a64e80106cf7.png)
 
+The contract ensures that amount in microalgos must be >=100, otherwise the transaction will be rejected.
+
+Transaction Pass:
+```bash
+algob> .editor
+// Entering editor mode (Ctrl+D to finish, Ctrl+C to cancel)
+// Transactions for Transaction for ALGO - Contract : '2-gold-contract-asc.teal'  (Contract Mode)
+// sender is contract account
+const algoTxParam = {
+  type: rtypes.TransactionType.TransferAlgo,
+  sign: rtypes.SignType.LogicSignature,
+  fromAccountAddr: lsig.address(),
+  toAccountAddr: john.addr,
+  amountMicroAlgos: 20n, // amt < 100
+  lsig: lsig,
+  payFlags: { totalFee: 1000 }
+};
+
+// Transaction PASS - As according to .teal logic, amount should be <= 100
+algob.executeTransaction(deployer, algoTxParam);
+
+{
+  'confirmed-round': 4418,
+  'pool-error': '',
+  'receiver-rewards': 93,
+  'sender-rewards': 1,
+  txn: {
+    lsig: { l: [Uint8Array] },
+    txn: {
+      amt: 20,
+      fee: 1000,
+      fv: 4416,
+      gen: 'private-v1',
+      gh: [Uint8Array],
+      lv: 5416,
+      rcv: [Uint8Array],
+      snd: [Uint8Array],
+      type: 'pay'
+    }
+  }
+}
 ```
-#Transaction PASS (amount <= 100)
-await algob.transferMicroAlgosLsig(deployer, { addr: sender}, johnAccount.addr, 20, lsig);
 
-#Transaction FAIL (amount should be <= 100)
-await algob.transferMicroAlgosLsig(deployer, { addr: sender}, johnAccount.addr, 200, lsig);
+Transaction fail:
+```bash
+algob> .editor
+// Entering editor mode (Ctrl+D to finish, Ctrl+C to cancel)
+const invalidTxnParams = {
+  type: rtypes.TransactionType.TransferAlgo,
+  sign: rtypes.SignType.LogicSignature,
+  fromAccountAddr: lsig.address(),
+  toAccountAddr: john.addr,
+  amountMicroAlgos: 200, // amt > 100
+  lsig: lsig,
+  payFlags: { totalFee: 1000 }
+};
+
+// Transaction FAIL - Gets rejected by logic - As according to .teal logic, amount should be <= 100
+algob.executeTransaction(deployer, invalidTxnParams);
+
+// rejected by logic
```

Comment:
![Screenshot from 2021-06-22 09-34-23](https://user-images.githubusercontent.com/26249903/122861448-1d1f7a00-d33d-11eb-8790-6390fcd31af0.png)

i think this error needs to be updated, i got this error. `Bad Request` is not much clear also.

Reply:
i posted the error i got in the session (without editing anything). I don't think we should edit the response. note that this is a code snippet/snapshot

Reply:
why it is different than this ? it is what i got

Reply:
`without editing`

Reply:
because account state is different in different networks. I got a very big error message, also the screenshot you shared only displays the bottom. 

I would've been a very large snippet otherwise, main goal is to display transaction was rejected. Feel free to share your response, if it's shorter, i will replace this one.

Reply:
that means you have edited the response. no ?

Reply:
and you wrote `you will get something like this`

Reply:
yes main goal is `rejected by logic` and not `BAD REQUEST` which is very unclear and also i don't how you got it.

Reply:
leaving something like `...` is pretty obvious to anyone that we're trying to avoid messing up leaving a very large  response.

> yes main goal is rejected by logic and not BAD REQUEST which is very unclear and also i don't how you got it.

The message starts from bad request. as said above, could you just share the entire code snippet ? I will replace this one with yours if it's shorter.

Reply:
no point in discussing this, do as you wish